src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/LicenseAcceptanceWindow.xaml.cs

```diff
@@ -39,7 +41,9 @@ private void ViewLicense_Click(object sender, RoutedEventArgs e)
                         DataContext = licenseFile
                     };
 
-                    licenseFile.LoadLicenseFile();
+                    //TODO: plumb identity in
```

Comment:
URL for tracking issue please

Reply:
will fix as part of this PR. thx for noticing.